curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -485,7 +485,6 @@ private String protectedPathInForeground(String adjustedPath, byte[] data) throw
         }
         catch ( Exception e)
         {
-            ThreadUtils.checkInterrupted(e);
```

Comment:
No, because the `InterruptedException` is rethrown in every case. Thread interruption status is represented by _either_ the thread interrupt status _or_ an `InterruptedException`, not both. This is mentioned in the Java tutorials at http://docs.oracle.com/javase/tutorial/essential/concurrency/interrupt.html:

> By convention, any method that exits by throwing an InterruptedException clears interrupt status when it does so.